Backend/Routes/users.js

```diff
@@ -807,7 +805,8 @@ router.patch("/meetings/:meetingId", async (req, res) => {
       return res.status(404).json({ error: "Meeting not found" });
     }
 
-    meeting.status = status;
+    if (status) meeting.status = status;
```

Comment:
nit: I would avoid inline if statements, it's still better to have curly brackets